blackbird_python/blackbird/error.py

```diff
@@ -62,8 +63,9 @@ def __init__(self, msg):
         Args:
             msg (str): message to be printed in the traceback
         """
+        #pylint: disable=super-init-not-called,unused-argument
```

Comment:
`super-init-not-called`: that seems like an overly strict thing to check for

Reply:
Yeah, this is the first time I even noticed it was a thing pylint checks